File: src/targets/python/requests/fixtures/nested.py
import requests

<<<<<<< HEAD
url = "https://httpbin.org/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value"

response = requests.get(url)

print(response.text)
=======
url = "http://mockbin.com/har"

querystring = {"foo[bar]":"baz,zap","fiz":"buz","key":"value"}

response = requests.get(url, params=querystring)

print(response.json())
>>>>>>> upstream/master

File: src/targets/clojure/clj_http/fixtures/nested.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/get "https://httpbin.org/anything" {:query-params {:foo[bar] "baz,zap"
                                                           :fiz "buz"
                                                           :key "value"}})
=======
(client/get "http://mockbin.com/har" {:query-params {:foo[bar] "baz,zap"
                                                     :fiz "buz"
                                                     :key "value"}})
>>>>>>> upstream/master
